README.md

```diff
@@ -90,7 +90,7 @@ Once installed, Semgrep can be run with single rule patterns or entire rule sets
 $ semgrep -e '$X == $X' --lang=py path/to/src
 
 # Run a ruleset with rules for many languages
-$ semgrep --config=https://semgrep.dev/p/r2c-CI path/to/src
+$ semgrep --config=p/r2c-CI path/to/src
```

Comment:
@brendongo We're standardizing on lowercase policy/rulesets in documentation. Up for adjusting this?